Terminal.Gui/Views/Shortcut.cs

```diff
@@ -626,10 +626,17 @@ private void UpdateKeyBinding ()
             // Disable the command view key bindings
             CommandView.KeyBindings.Remove (Key);
             CommandView.KeyBindings.Remove (CommandView.HotKey);
-            KeyBindings.Remove (Key);
-            KeyBindings.Add (Key, KeyBindingScope | KeyBindingScope.HotKey, Command.Accept);
 
-            //KeyBindings.Add (Key, KeyBindingScope.HotKey, Command.Accept);
+            if (KeyBindingScope.FastHasFlags (KeyBindingScope.Application))
```

Comment:
The `UpdateKeyBinding` method isn't updating nothing because in practice it's adding only the new `Key` value but doesn't remove the previous `Key` value, ending up with a shortcut with more than one key if their value are updated.

Reply:
This unit tests will fail:

```cs
    [Fact]
    [AutoInitShutdown]
    public void Changing_Shortcut_Key_Removes_Previous_Key ()
    {
        var newActionCount = 0;

        var statusBar = new StatusBar (
                                       new []
                                       {
                                           new Shortcut (Key.N.WithCtrl, "New", () => newActionCount++)
                                       });
        var top = new Toplevel ();
        top.Add (statusBar);
        var rs = Application.Begin (top);

        Assert.Equal (0, newActionCount);
        Assert.True (Application.OnKeyDown (Key.N.WithCtrl));
        Assert.False (Application.OnKeyDown (Key.W.WithCtrl));
        Assert.Equal (1, newActionCount);

        ((Shortcut)statusBar.Subviews [0]).Key = Key.W.WithCtrl;
        Assert.False (Application.OnKeyDown (Key.N.WithCtrl));
        Assert.True (Application.OnKeyDown (Key.W.WithCtrl));
        Assert.Equal (2, newActionCount);

        Application.End (rs);
        top.Dispose ();
    }

```